src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -2754,6 +2787,46 @@ protected function castAttribute($key, $value)
 	}
 
 	/**
+	 * Decrypt the given value.
+	 *
+	 * @param  string  $value
+	 * @return mixed
+	 *
+	 * @throws \Exception
+	 */
+	protected function decrypt($value)
+	{
+		if (is_null($value)) return null;
```

Comment:
@JosephSilber Are you sure? Because a search for `return null;` over the whole framework yields no results at all.